drivers/slipdev/include/slipdev_params.h

```diff
@@ -34,9 +34,13 @@ extern "C" {
  */
 #ifndef SLIPDEV_PARAM_UART
 # ifndef MODULE_SLIPDEV_STDIO
-#  define SLIPDEV_PARAM_UART        (UART_DEV(0))
+#  ifdef MODULE_USBUS_CDC_ACM
+#   define SLIPDEV_PARAM_UART       UART_DEV(0)
+#  else
+#   define SLIPDEV_PARAM_UART       UART_DEV(1)
+#  endif
```

Comment:
The reasoning here is non-obvious to me here. If there is a reason this is a sane default, please add a comment.

Is it about avoiding `STDIO_UART_DEV` for when `MODULE_SLIPDEV_STDIO` is not in use? Some boards do use `UART_DEV(1)` for stdio, others use `UART_DEV(0)`.

Reply:
> Is it about avoiding `STDIO_UART_DEV` for when `MODULE_SLIPDEV_STDIO` is not in use? 

Yes, I copied that logic from `ethos`

> Some boards do use `UART_DEV(1)` for stdio, others use `UART_DEV(0)`

True, but 99% use `UART_DEV(0)`, so the current default is broken in 99% of all cases whereas the new default might only be broken in 1% of all cases :wink:

Reply:
If it's controversial I'm fine with dropping it from the PR, just thought it would be a nice convenience improvement.

Reply:
No, let's have it consistent with ethos. I think I have an idea to also be a good default for the 1% of the cases, but that should be done for both as follow up.